tools/istio-clean-iptables/pkg/cmd/testdata/kube-virt-interfaces.golden

```diff
@@ -0,0 +1,56 @@
+iptables -t PREROUTING -D nat -i eth1 -j ISTIO_REDIRECT
```

Comment:
Oh weird, I wasn't really aware we had a `kube-virt` specific override flag for this.

I guess we're stuck with a jump-per-interface.